tests/giftcard/dropin-generic-giftcard.spec.js

```diff
@@ -46,4 +50,25 @@ async function enterGiftcardDetails(page) {
     const redeemButton = page.locator('.adyen-checkout__button__text >> visible=true');
     await expect(redeemButton).toBeVisible();
     await redeemButton.click();
+}
+
+async function enterSchemeDetails(page) {
+
+    // Wait for network state to be idle
+    await page.waitForLoadState('networkidle')
+
+    // Find iframe and fill "Card number" field
+    const cardNumberFrame = page.frameLocator('internal:attr=[title="Iframe for secured card number"i]');
+    await cardNumberFrame.getByPlaceholder('1234 5678 9012 3456').fill(utilities.CARD_NUMBER);
```

Comment:
The usage of utility.js here is only to retrieve the static strings, it defeats purpose of having the utility function itself. What do you think, should we extract this logic into "enterGiftCardDetails()" in utilities.js?

We cannot reuse the existing functions `fillComponentCardDetails()`/`fillDropinCardDetails()` logic from [utilities.js](https://github.com/adyen-examples/adyen-testing-suite/blob/main/tests/utilities.js#L33) (e.g. 'internal:attr=[title="Iframe for secured gift card number"i]' vs. 'iframe[title*="card number"]' as they are targetting different frames.